src/main/java/uk/gov/digital/ho/hocs/queue/complaints/ukvi/UKVIComplaintService.java

```diff
@@ -1,16 +1,20 @@
 package uk.gov.digital.ho.hocs.queue.complaints.ukvi;
 
+import com.fasterxml.jackson.databind.ObjectMapper;
 import lombok.extern.slf4j.Slf4j;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.beans.factory.annotation.Value;
 import org.springframework.stereotype.Service;
 import uk.gov.digital.ho.hocs.application.ClientContext;
+import uk.gov.digital.ho.hocs.domain.repositories.EnumMappingsRepository;
 import uk.gov.digital.ho.hocs.queue.complaints.ComplaintService;
 
 @Slf4j
 @Service
 public class UKVIComplaintService {
 
+    private final ObjectMapper objectMapper;
+    private final EnumMappingsRepository enumMappingsRepository;
```

Comment:
I've checked and this is the right place to specify this - but this is annoying that this has to be at this high level and passed down so much...